src/infer_response.cc

```diff
@@ -58,6 +60,12 @@ InferResponse::OutputTensors()
   return output_tensors_;
 }
 
+std::string&
```

Comment:
Should it be mutable or `const`?

Reply:
It should be `const`. Updated [Mark response parameters accessor const and JSON serializable](https://github.com/triton-inference-server/python_backend/pull/394/commits/f23114704d0a6135962b72ccb88b55425c48b5b0)